app/decorators/user_decorator.rb

```diff
@@ -121,6 +121,10 @@ def recent_events
     (events + identities).sort { |thing_a, thing_b| thing_b.happened_at <=> thing_a.happened_at }
   end
 
+  def identities
+    user.identities.order('last_authenticated_at DESC').map(&:decorate)
```

Comment:
Thoughts on adding a limit to this? Could get out of hand if the user has like 20 SPs they've signed into? We may want to show the last n identities instead of all of them. (cc @mkhandekar)

Also, you can use `.order(last_authenticated_at: :desc)` which does the same thing but imo is a little neater. That's an opinion though, so ultimately up to you.